lib/bloc/present_signed_document_cubit.dart

```diff
@@ -41,11 +45,10 @@ class PresentSignedDocumentCubit extends Cubit<PresentSignedDocumentState> {
     File? file;
 
     try {
-      file = await _getTargetFile();
-      final bytes = await Future.microtask(
-        () => base64Decode(signedDocument.content),
-      );
-      await file.writeAsBytes(bytes);
+      file = await _getTargetPath().then((path) => File(path));
+      // TODO Catch and still allow sharing

```

Comment:
Čiže zatiaľ je tu fix taký, že sa skúsi vytvoriť Downlaods folder. A ak to vytváranie padne, tak sa nezobrazí ani share button?

Reply:
Ten comment je mierne zavadzajuci.

Sharovat sa bude dat tak ci tak, to je vidno v Screen-e switch na state, error case.